frontend/src/scenes/data-warehouse/settings/source/Syncs.tsx

```diff
@@ -88,14 +88,14 @@ export const Syncs = ({ id }: SyncsProps): JSX.Element => {
                                       hideInstanceIdColumn={true}
                                       defaultFilters={{
                                           instanceId: job.workflow_run_id,
-                                          dateFrom: dayjsUtcToTimezone(job.created_at, timezone)
-                                              .add(-1, 'day')
-                                              .toISOString(),
+                                          dateFrom: dayjsUtcToTimezone(job.created_at, timezone).format(
+                                              'YYYY-MM-DD HH:mm:ss'
+                                          ),
                                           dateTo: job.finished_at
-                                              ? dayjsUtcToTimezone(job.finished_at, timezone)
-                                                    .add(1, 'day')
-                                                    .toISOString()
-                                              : dayjs().add(1, 'day').toISOString(),
+                                              ? dayjsUtcToTimezone(job.finished_at, timezone).format(
```

Comment:
I'd still want a small buffer here. The finished at timestamp isn't perfectly aligned for when our workflow actually terminated